app/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    buildToolsVersion "21.1.1"
```

Comment:
Please use 21.1.2. Anything else is breaking the Travis CI build: https://travis-ci.org/intrications/intent-intercept/builds/51622311